piecash/core/session.py

```diff
@@ -251,7 +251,7 @@ def create_book(
 
     :param str sqlite_file: a path to an sqlite3 file (only used if uri_conn is None)
     :param str uri_conn: a sqlalchemy connection string
-    :param str currency: the ISO symbol of the default currency of the book
+    :param str currency: the ISO symbol of the default currency of the book, "locale" returns local international currency symbol
```

Comment:
could you also replace the current default of "EUR" in the function signature (line 237) with "locale". I think it is a better default, no ?